views/pug/index.pug

```diff
@@ -1,7 +1,7 @@
 html
   head
     title FCC Advanced Node and Express
-    meta(name='description', content='A cool thing made with Glitch')
+    meta(name='description', content='Home page')
```

Comment:
this bypasses one of the tests to return the object with this text.  it's inconsistent throughout the coursework though and causes a lot of confusion.

Reply:
It is not so much a bypass, because the new lessons do not ask the user to pass the object with `'Home Page'`. I changed the tests around. So, instead this is only used by these lessons:
1) Create New Middleware
2) Logging a User Out
3) Registration of New Users

There are many ways this could have been tested for. Personally, I prefer this method which does not rely on the user's implementation (spelling) of the previous lesson.